### ISSUE TITLE
Figure out how to support /milestone for kubernetes-sigs repositories

### ISSUE THREAD

**Issue by roberthbailey** [kubernetes/test-infra#8690]
Right now you have to be in the the `<org>/kubernetes-milestone-maintainers` github team to set a milestone on an issue via the `/milestone` command. For the main repo, this seems fine. For repositories underneath kubernetes-sigs this isn't going to work. The repositories in that org are all owned and maintained by different SIGs. Having a single github group that sets milestones for all of them doesn't make sense. Each repo should have a configurable way to set who can use the milestone command (without needing to make contributors editors of the repository). 

/cc @neolit123 @timothysc

**Comment by neolit123** [comment]
i think having this more flexible outside of `k/k` makes sense.

> Each repo should have a configurable way to set who can use the milestone command (without needing to make contributors editors of the repository).

perhaps the easiest solution is to allow all regular org members to be able to use the command, but only outside of `k/k`. this could create some noise, but i wonder if that's going to be that much of a problem.

**Comment by stevekuznetsov** [comment]
Wasn't this already implemented in 19b225c9d44c8d2390317cc7cf46a4c101c10703 by @amwat ?

**Comment by stevekuznetsov** [comment]
Perhaps needs some better doc 
/kind documentation

**Comment by neolit123** [comment]
> Wasn't this already implemented in 19b225c 

https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md

```
+    # You can curl the following endpoint in order to determine the github ID of your team
+    # responsible for maintaining the milestones:
+    # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
```

so it's already supported. nice.
